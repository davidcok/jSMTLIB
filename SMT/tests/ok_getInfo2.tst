; get-info that depends on check-sat
(set-logic QF_UF)
(assert true)
(check-sat)
(get-option :all-statistics)
(get-option :reason-unknown)

