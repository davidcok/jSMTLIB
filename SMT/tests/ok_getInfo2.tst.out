success
success
sat
unsupported
unsupported
